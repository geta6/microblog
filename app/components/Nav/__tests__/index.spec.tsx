import React from 'react';
import renderer from 'react-test-renderer';
import { NavComponent } from '..';

describe('NavComponent', () => {
  test('snap shot', () => {
    const component = renderer.create(<NavComponent />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
