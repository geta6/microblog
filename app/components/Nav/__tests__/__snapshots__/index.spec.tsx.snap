// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavComponent snap shot 1`] = `
<div
  className="style__Container-u8fa6-0 kpccir"
>
  <div
    className="style__Col-u8fa6-1 imLpjy"
  >
    <div
      className="style__Logo-u8fa6-2 eJIrdo"
    >
      Microblog
    </div>
    <div
      className="style__Row-u8fa6-3 fJYSMW"
    >
      Menu1
    </div>
    <div
      className="style__Row-u8fa6-3 fJYSMW"
    >
      Menu2
    </div>
    <div
      className="style__Row-u8fa6-3 fJYSMW"
    >
      Menu3
    </div>
  </div>
</div>
`;
